Script/update-testutils-version.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
```

Comment:
Instead of this file,  can you update `update-version.sh` with the logic for TestUtils? It should only be a one line change.